scripts/make_tor_targets.py

```diff
@@ -0,0 +1,114 @@
+import json
```

Comment:
This script should go in the API package with its dedicated .timer and .service 
With logging and metrics similar to the API so that we can monitor the run and alarm when needed.
The 2 .json files can be removed from here. The API runner would start without .json files and download the first copy.